src/ofdm.c

```diff
@@ -942,8 +942,9 @@ int ofdm_sync_search(struct OFDM *ofdm, COMP *rxbuf_in) {
      * so it is primed for when we have to call ofdm_demod()
      */
 
-    memcpy(&ofdm->rxbuf[0], &ofdm->rxbuf[ofdm->nin],
-        (ofdm_rxbuf - ofdm->nin) * sizeof (complex float));
+    /* note can't use memcpy when src and dest overlap */
+    memmove(&ofdm->rxbuf[0], &ofdm->rxbuf[ofdm->nin],
+           (ofdm_rxbuf - ofdm->nin) * sizeof (complex float));
```

Comment:
@srsampson Had some sync issues on 2020, and noticed use of memcpy on overlapping buffers/. Just happened to hit this one at $dayjob recently.  Subtle

Reply:
I think memcpy should be deprecated throughout the repository in favor of memmove. My thinking is that these two functions are optimized in the compiler, and will be faster than a for loop, but I've never actually looked at the output code.

Reply:
It's a very subtle one .... I've used it without considering overlap for years.  Took me weeks to find it at $dayjob, as it would only come up on occasional Travis runs.  I figure just use for loops.  There aren't many memory copies that are significant in terms of CPU on modern machines.

memmove adds another buffering step so it's not great either.